kolibri/content/serializers.py

```diff
@@ -48,9 +49,15 @@ def get_ancestors(self, target_node):
         """
         return target_node.get_ancestors().values('pk', 'title')
 
+    def get_thumbnail(self, target_node):
+        try:
+            return target_node.files.get(thumbnail=True, available=True).get_url()
```

Comment:
You get the first one according to database order.

![image](https://cloud.githubusercontent.com/assets/1680573/16702241/a1d67f12-4519-11e6-9f47-fdbebf8aca57.png)